contracts/staking/StakingInfo.sol

```diff
@@ -397,28 +400,24 @@ contract StakingInfo is Ownable {
             uint256 activationEpoch,
             uint256 deactivationEpoch,
             address signer,
-            uint256 _status
+            IStakeManagerLocal.Status status
         )
     {
         IStakeManagerLocal stakeManager = IStakeManagerLocal(
             registry.getStakeManagerAddress()
         );
-        address _contract;
-        IStakeManagerLocal.Status status;
+        uint256 delegatedAmount;
         (
             amount,
             reward,
             activationEpoch,
             deactivationEpoch,
             ,
             signer,
-            _contract,
-            status
+            ,
+            status, , , ,delegatedAmount,
         ) = stakeManager.validators(validatorId);
-        _status = uint256(status);
-        if (_contract != address(0x0)) {
-            reward += IStakeManagerLocal(_contract).validatorRewards();
-        }
+        reward = reward.add(delegatedAmount);
```

Comment:
Is `delegatedAmount` the same as `reward`? `delegatedAmount` is total delegation for that user, right?

Reply:
I am not sure why is it here. Fixed.